administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -82,6 +82,11 @@
 $isModal = $input->get('layout') == 'modal' ? true : false;
 $layout = $isModal ? 'modal' : 'edit';
 $tmpl = $isModal ? '&tmpl=component' : '';
+
+JHtml::script('system/share.js', false, true);
+JFactory::getDocument()->addScriptDeclaration('
+ 	var sharebutton_url = "' . addslashes(JUri::base()) . 'index.php?option=com_content&task=article.shareDraft&format=json&' . JSession::getFormToken() . '=1";
+  ');
```

Comment:
@roland-d @yvesh , Unable to store the values in table despite of table being present. What do you suggest?